tests/server/test_agents_api.py

```diff
@@ -118,12 +118,12 @@ def agent_names(agents: list[dict[str, str]]) -> list[str]:
     return [agent["name"] for agent in agents]
 
 
-@pytest.mark.parametrize("order_by_column", ["name", "status", "process_name", "last_failover", "paused"])
```

Comment:
if would be  interesting to make only the absolute minimal amount of modification to this testcase, so we know we don't break the front-end